src/app/features/genes/components/gene-druggability/gene-druggability.component.ts

```diff
@@ -43,28 +43,23 @@ export class GeneDruggabilityComponent implements OnInit {
     private geneService: GeneService
   ) {}
 
-  ngOnInit() {
-    //
-  }
-
   init() {
     // Update the initial buckets
     if (this.gene) {
       if (!this.gene.druggability) {
-        this.gene.druggability = [
-          {
-            sm_druggability_bucket: this.getDefaultBucketNumber(),
-            safety_bucket: this.getDefaultBucketNumber(),
-            abability_bucket: this.getDefaultBucketNumber(),
-            pharos_class: '',
-            classification: this.getDefaultText(),
-            safety_bucket_definition: this.getDefaultText(),
-            abability_bucket_definition: this.getDefaultText(),
-          },
-        ];
+        this.gene.druggability =
+        {
+          sm_druggability_bucket: this.getDefaultBucketNumber(),
+          safety_bucket: this.getDefaultBucketNumber(),
+          abability_bucket: this.getDefaultBucketNumber(),
+          pharos_class: [],
+          classification: this.getDefaultText(),
+          safety_bucket_definition: this.getDefaultText(),
+          abability_bucket_definition: this.getDefaultText(),
+        };
```

Comment:
Leaving the removed fields here seems harmless since they just get the defaults and we don't display those anywhere, but it woiuld be nice if they were gone entirely in the monorepo version of the app.